core/services/chainlink/testdata/config-invalid.toml

```diff
@@ -159,6 +159,11 @@ APIKey = 'key'
 
 [[Starknet]]
 
+[[Aptos]]
+Enabled = 1
+
+[[Aptos]]
```

Comment:
What is this extra one for?

Reply:
Looking at the other Non-evm config, it seems like its to tests whether the config becomes invalidated without a config. Thought it might be testing the edge case that's why i added it in 


https://github.com/smartcontractkit/chainlink/blob/3f8c00a6f1884f765bbe9e4b70e0dc4fb94a0088/core/services/chainlink/testdata/config-invalid.toml#L137-L146

Reply:
Is there a corresponding failing test case? If not, then it can be removed.

Reply:
There isn't a corresponding failing test case for now, removed it. thanks for the suggestion